### ISSUE TITLE
Problem accessing to projects

### ISSUE THREAD

**Issue by jademlc** [doccano/doccano#926]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10

### Describe the problem
When we want to create a new project, and other admins or anotators are already in the users, the new project doesn't show in their accounts.
We had to delete the other users and add them to the project again for them to have access.

**Comment by icoxfog417** [comment]
Please provide your environment information following to the [ISSUE_TEMPLATE](https://github.com/doccano/doccano/blob/master/.github/ISSUE_TEMPLATE.md)

**Comment by Hironsan** [comment]
I want to know what is the ideal situation and how to reproduce the problem. It's necessary for the project administrator to add some users to the project to allow them to show the project.

**Comment by jademlc** [comment]
When accessing Doccano for the first time, we added two other admins via the admin page. Then, we created a new project and the two other admins couldn't access it, even though they were on the list of the project's users. We had to delete them from the project and add them again. 
I don't know if the same problem occurs if we add users.

I use Doccano in Google Chrome via Windows 10 and my python version is 3.8.5
